chapters/operatorsandexpressions.tex

```diff
@@ -36,53 +36,52 @@ \section{Operator Precedence and Associativity}\label{operator-precedence-and-as
 an expression. An operator with higher precedence is evaluated before an
 operator with lower precedence in the same expression.
 
-The following table presents all the expression operators in order of
-precedence from highest to lowest, as derived from the Modelica grammar
-in \cref{modelica-concrete-syntax}. All operators are binary except the postfix operators and
-those shown as unary together with \emph{expr}, the conditional
-operator, the array construction operator \lstinline!{ }! and concatenation
-operator \lstinline![ ]!, and the array range constructor which is either binary
-  or ternary. Operators with the same precedence occur at the same line of
-the table:
+The following table presents all the expression operators in order of precedence.
```

Comment:
@HansOlsson, this commit needs special attention since it changes things a bit in the _Operator group_ column to simplify the table by having fewer things in each LaTeX row, and using the horizontal lines to show separation between the precedence groups.